modules/objc/examples/code_snippets/SampleCodeTest.m

```diff
@@ -1804,7 +1804,48 @@ - (void) dontTestQuerySyntaxJson {
 
 // end::query-syntax-pagination[]
 
+- (void) docsonly_QuerySyntaxN1QL() {
+    /* Documentation Only query-syntax-n1ql
+        Declared elsewhere: CBLDatabase* argDb
+    */
 
+    // tag::query-syntax-n1ql[]
+    NSError *error;
+
+    CBLDatabase *db == argDB;
```

Comment:
do you think this two variable declaration are important, and need to be displayed?

Reply:
DONe